research/src/pages/component-spec-template.mdx

```diff
@@ -0,0 +1,213 @@
+> This document represents a component specification template. Its purpose
+> is to provide a point-in-time guide as to how a component should be
+> authored, in light of common anatomies, behaviors, and accessibility.
+> The primary audience is a developer authoring a given component.
+
+# {ComponentName} Component Specification
+
+## Overview
+
+*The name of the component, along with a high-level description.*
+
+### Background
+
+*Relevant historical or background information, related existing issues, etc.*
+
+### Use Cases
+
+*Primary use cases for this component.*
+
+### Non-goals
+
+*A list of use cases, features, or functionality which are **not** goals for the component.*
+  
+### Features
+
+*A list of the key features unique to this component.*
+
+### Risks and Challenges
```

Comment:
I can certainly think of risks for various components. For example, if there's a component that relates to a web standard that's in flux. There's a risk there. Something like a select or list control might have a technical risk related to handling large sets of data, etc. I find it's worth it to at least think through these sorts of things.

Reply:
@EisenbergEffect yeah there are definately risks for any component or it's sub features and am not stating that they shouldn't be thought out. That's precisely what issues are for which is to track things that are blocking a standard such as perf for large datasets, etc. Normally, in other specs these issues are linked inline to the area that has the Issue. I'd probably be more in favor of simply changing this to Issues and possibly having the important ones inline to their items that have the issue (here's an [example](https://drafts.csswg.org/css-color-4/#resolving-color-values) in BikeShed). This could then link to said issues. If there are challenges as it relates to specific sections of implementation then I think those should be noted inline so the implementor isn't having to go from implementation text to challenges that may/may not exist (it would be similar to comments in review not being inlined but just a combination at the bottom that you then had to join up with the code yourself). Again, I wouldn't block on this.

Reply:
I've changed some of the wording to remove anything related to implementation. I've made this into general risks/challenges and added a note about linking issues for platform blockers.